src/eterna/mode/PoseEdit/PoseEditMode.ts

```diff
@@ -1269,140 +1270,11 @@ export default class PoseEditMode extends GameMode {
                 this._puzzle.getBarcodeHairpin(Sequence.fromSequenceString(seq)).sequenceString()
             ));
 
-        this._scriptInterface.addCallback('current_folder', (): string | null => (
```

Comment:
You should go commit by commit, since the first commit doesn't modify this code, only moves it (and adds code around it)